Realm/RLMSchema.mm

```diff
@@ -126,12 +126,20 @@ - (instancetype)init {
 
 - (NSArray *)objectSchema {
     if (!_objectSchema) {
+        if (self == s_sharedSchema) {
+            // Ensure the schema is fully initialized
+            [RLMSchema sharedSchema];
```

Comment:
This initialization is incorrect. We specifically go out of our way to avoid fully initializing the shared schema when it's obtained via `partialSharedSchema` because on watchOS calling `objc_copyClassList()` can easily exceed the memory limit.